src/bin/ion/commands/generate/generator.rs

```diff
@@ -1085,16 +1095,13 @@ mod isl_to_model_tests {
         );
         let data_model_node = java_code_generator.convert_isl_type_def_to_data_model_node(
             &"my_struct".to_string(),
-            FieldPresence::Required,
             &isl_type,
             &mut CodeGenContext::new(),
             false,
         )?;
         let abstract_data_type = data_model_node.code_gen_type.unwrap();
         assert_eq!(
-            abstract_data_type
-                .fully_qualified_type_ref::<JavaLanguage>()
-                .unwrap(),
+            abstract_data_type.fully_qualified_type_ref::<JavaLanguage>(),
```

Comment:
(🗺️ PR tour) All the following changes in this file are because of changing `fully_qualified_type_ref` return `FullyQualfiiedTypeReference` instead of `Option<FullyqualifiedTypeReference.`